internal/host/host_test.go

```diff
@@ -1323,6 +1323,72 @@ var _ = Describe("UpdateInventory", func() {
 			Expect(h.InstallationDiskID).To(Equal(diskId))
 		})
 	})
+	Context("Interfaces", func() {
+		const (
+			diskName = "FirstDisk"
+			diskId   = "/dev/disk/by-id/FirstDisk"
+		)
+		var (
+			physicalInterface = models.Interface{
```

Comment:
This doesn't look like it is used anywhere other than where we assert on the type.
Couldn't we just hardcode "physical" in the test like we're doing everywhere else? Why define the whole structure?

Reply:
Removed it, thank you!